packages/ui/package.json

```diff
@@ -11,41 +11,43 @@
         "lint": "TIMING=1 eslint --ext .tsx,.ts src/"
     },
     "devDependencies": {
-        "@babel/plugin-proposal-private-methods": "^7.18.6",
+        "@babel/plugin-proposal-class-properties": "7.18.6",
```

Comment:
Packages added/changed as part of this PR are:
- `@babel/plugin-proposal-class-properties`
- `@ethersproject/bignumber`
- `ethers`